example/t3dn_bip_example/previews.py

```diff
@@ -3,17 +3,26 @@
 
 folder = None
 collection = None
```

Comment:
Do you still use this collection?

Reply:
For KIT OPS, we use a single collection for the whole addon; so I think a simple example can definitely do the same.
No need for this dictionary approach.

Reply:
Uhm, so I've gotten rid of the dictionary `PREVIEW_COLL` and resorted to using `collection` instead.